rules/duo_rules/duo_user_anomalous_push.yml

```diff
@@ -0,0 +1,40 @@
+AnalysisType: rule
+Filename: duo_user_anomalous_push.py
+RuleID: DUO.User.Denied.AnomalousPush
+DisplayName: Duo User Auth Denied For Anomalous Push
+Enabled: true
+DedupPeriodMinutes: 15
+LogTypes:
+  - Duo.Authentication
+Tags:
+  - Duo
+Severity: Medium
+Description: >
+  A Duo authentication was denied due to an anomalous 2FA push.
+Reference: https://duo.com/docs/adminapi#authentication-logs
+Runbook: Follow up with the user to confirm they intended several pushes in quick succession.
+Tests:
+  -
+    Name: anomalous_push_occurred
```

Comment:
please provide a test case where the ExpectedResult is false

Reply:
Done 👍